sklearn/utils/_estimator_html_repr.py

```diff
@@ -78,24 +79,52 @@ def _write_label_html(
     outer_class="sk-label-container",
     inner_class="sk-label",
     checked=False,
+    url_link="",
```

Comment:
What was the thinking around using `""` (empty string) as the "not set" value instead of `None`? The latter seems more usual?! Commenting here but the comment also applies to `_get_url_link`/this whole PR.

Reply:
The main reason is that `url_link` is used directly in f-strings, so when it is not set (for any reason), then the final html doesn't need any changes, while with `None` we'd have to then change it back to `""` anyway for the final html string. I think it's simpler to have `""` here.